infrastructure/proto/log_replication_metadata.proto

```diff
@@ -129,6 +129,7 @@ message ReplicationEvent {
 
   enum ReplicationEventType {
     FORCE_SNAPSHOT_SYNC = 0;
+    ROLLING_UPGRADE_FORCE_SNAPSHOT_SYNC = 1;
```

Comment:
can you rename it to something like - UPGRADE_COMPLETION_FORCE_SNAPSHOT_SYNC?  Basically, a snapshot sync must be triggered after upgrade completion, rolling or not.